__tests__/workers/newNotificationV2Mail.ts

```diff
@@ -2813,7 +2813,7 @@ describe('warm_intro notification', () => {
       .calls[0][0] as SendEmailRequestWithTemplate;
 
     expect(args.message_data).toEqual({
-      title: `It's a match!`,
+      title: `[Action Required] It's a match!`,
```

Comment:
This should not be here?

Reply:
i pushed a change to main earlier, didn't notice we test for the title. i actually looked for it but missed this place. so i'm fixing it here